hazelcast-jet-core/src/test/java/com/hazelcast/jet/pipeline/test/TestSourcesTest.java

```diff
@@ -48,6 +49,30 @@ public void test_items() {
         jet().newJob(p).join();
     }
 
+    @Test
+    public void test_longStream() throws Throwable {
+        int itemsPerSecond = 10;
+        int timeout = 10;
+        int numberOfExpectedValues = timeout * itemsPerSecond;
+        Long[] input = LongStream.range(0, numberOfExpectedValues).boxed().toArray(Long[]::new);
```

Comment:
why not `.collect(toList())` directly?

Reply:
Done with 27522575900e3e43d31d1615b99d225ea8025c35